packages/formula-editor/src/formula-editor.ts

```
@@ -0,0 +1,319 @@
+    this.content = this._parser.addParentheses(this.content) ?? this.content;
+    this.parseInput();
+
+    this._calculatedResult = calculatedResult.result;
+    this.errorString = calculatedResult.errorString;
+
+    this._recommendations = null;
+    this.requestUpdate();
+  }
+
+  requestFormat() {
+    if(!Boolean(this.content)){
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)